src/columns.js

```diff
@@ -103,7 +102,9 @@ export class Columns {
             temp[3].push(d)
         })
 
+        // oops
```

Comment:
could you elaborate on "oops"?

Reply:
`headings` is an array consisting of `th` elements (header row cells) of the table.   I converted `headings` into a getter so that it would always reflect the current state of the table.  That means we can no longer modify it directly.  We have to actually update the table header itself, with `this.thead.appendChild`.  Then, that change will be reflected in `this.headings`.

So when I changed a few properties to getters, I looked through the code to make sure these properties were only being read from.  When I ran across that `this.headings.push(...)`, I realized that was a problem but didn't have time to fix it at the moment.  So I put that *oops* comment there to remind myself to get back to it and fix it.